ci-operator/config/monstorak/monstorak-operator/monstorak-monstorak-operator-master.yaml

```diff
@@ -0,0 +1,73 @@
+canonical_go_repository: github.com/monstorak/monstorak-operator
+
+base_images:
+  os:
+    cluster: https://github.com/monstorak/monstorak-operator
+    name: centos
+    namespace: openshift
+    tag: '7'
+  operator-registry:
+    name: "4.0"
+    namespace: ocp
+    tag: operator-registry
+
+resources:
+  '*':
+    limits:
+      memory: 2Gi
+    requests:
+      memory: 250Mi
+      cpu: 100m
+
+images:
+- from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/monstorak/monstorak-operator/out/operator 
+  dockerfile_path: openshift-ci/Dockerfile.deploy
+  to: monstorak
+- from: operator-registry
+  dockerfile_path: openshift-ci/Dockerfile.registry.intermediate
+  to: operator-registry-base
+- from: operator-registry-base
+  dockerfile_path: openshift-ci/Dockerfile.registry.build
+  to: monstorak-operator-registry
+# - from: src
+#   dockerfile_path: openshift-ci/Dockerfile.upgrade.build
+#   to: upgrade-manifest-build
+# - from: operator-registry-base
+#  inputs:
+#    upgrade-manifest-build:
+#      paths:
+#      - destination_dir: .
+#        source_path: /go/src/github.com/monstorak/monstorak-operator/out/manifests.tar.gz
+#  dockerfile_path: openshift-ci/Dockerfile.registry.build.next
+#  to: monstorak-operator-registry-next
+
+promotion:
+  cluster: https://github.com/monstorak/monstorak-operator
```

Comment:
???

Reply:
our cluster is `https://api.ci.openshift.org`

Reply:
Thanks @droslean , I have changed the cluster value. I was a bit confused with the value, thanks again for clearing it.
With the new PR, I have generated the files with latest 'ci-operator-prowgen'.

Reply:
Hi @droslean , I am not sure, why the errors/failures are happening. Regenerated all the files with latest 'ci-operator-prowgen'. Going through the error messages again...